src/features/donations/components/Donation/ShareDonation/ShareDonation.js

```diff
@@ -0,0 +1,72 @@
+import {
+  FacebookShareButton,
+  WhatsappShareButton,
+  TelegramShareButton,
+  TwitterShareButton,
+  RedditShareButton,
+  EmailShareButton,
+} from 'react-share';
+
+import { ReactComponent as WhatsappIcon } from 'assets/whatsapp_icon.svg';
+import { ReactComponent as FacebookIcon } from 'assets/facebook_icon.svg';
+import { ReactComponent as TelegramIcon } from 'assets/telegram_icon.svg';
+import { ReactComponent as TwitterIcon } from 'assets/twitter_icon.svg';
+import { ReactComponent as RedditIcon } from 'assets/reddit_icon.svg';
+import { ReactComponent as EmailIcon } from 'assets/email_icon.svg';
+import ShareButton from './ShareButton';
+
+const ShareDonation = () => {
+  const currentURL = window.location.href;
+  const description =
+    "Hola! hecha un vistazo de lo que estan donando en 'Che, ¿hoy donamos?'";

```

Comment:
```suggestion
  const description =
    "Hola! hecha un vistazo a esta donacion en 'Che, ¿hoy donamos?'";
```

@alejandroGonGon thoughts?

Reply:
I agree